server/routes/place/html.py

```diff
@@ -66,6 +67,49 @@
     'Continent',
 }
 
+# Location of manually written templates for SEO experimentation in GCS bucket
+SEO_EXPERIMENT_HTML_GCS_DIR = "seo_experiments/active"
+
+# Location of manually written templates for SEO experimentation on local disk
+SEO_EXPERIMENT_HTML_LOCAL_DIR = "config/seo_experiments/html_templates/active/"
+
+# Map DCID of SEO experiment places to their template filename
+SEO_EXPERIMENT_DCID_TO_HTML = {"country/EGY": "Egypt.html"}
+
+
+def get_seo_experiment_template(dcid: str) -> str:
+  """Load page template for SEO experiments
+  
+  Will load files from GCS unless server is running locally, in which case
+  will load from server/config/seo_experiments/html_templates/active instead.
+
+  If a file cannot be found or read, will return empty string instead.
+  """
+  experiment_template = ""
+  try:
+    filename = SEO_EXPERIMENT_DCID_TO_HTML[dcid]
+
+    if current_app.config['LOCAL']:
+      # Load template from local path
+      template_filepath = os.path.join(current_app.root_path,
+                                       SEO_EXPERIMENT_HTML_LOCAL_DIR, filename)
+    else:
+      # Load template from GCS
+      template_filepath = os.path.join("/tmp/seo_experiment/template/",
+                                       filename)
+      gcs.download_blob("datcom-website-config",
+                        os.path.join(SEO_EXPERIMENT_HTML_GCS_DIR, filename),
+                        template_filepath)
+
+    with open(template_filepath, 'r', errors='ignore') as f:
+      experiment_template = f.read()
+
+  except Exception as e:
+    logging.info(
```

Comment:
Should this be logging.error? I dont think we have a standard way to do logging-what do you think?

Reply:
+1 to using error. Will use error from here on out for exception/error catching.